Purchases.podspec

```diff
@@ -18,6 +18,8 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '9.0'
   s.osx.deployment_target = '10.12'
 
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
```

Comment:
yohoooo

Reply:
we should add an action item to update the installation guide to remove the `use_frameworks` thingy. because that's what this does right?